types/wordpress__components/autocomplete/index.d.ts

```diff
@@ -0,0 +1,131 @@
+import { ComponentType, ReactNode } from "@wordpress/element";
+
+declare namespace Autocomplete {
+    // FIXME: use `@wordpress/rich-text` value when it becomes available
```

Comment:
@DanielRosenwasser I'd like to merge (or rebase?) master into this branch so that I can fix this now that the PR for `rich-text` has merged. 

Any tips on how I should approach doing that? Or should I just wait for this to merge and send another PR with the update?